comicScraper.py

```diff
@@ -95,16 +102,15 @@ def extractImageUrlFromText(text):
     urlStart = text.find("https")
     return text[urlStart:urlEnd+5]
 
-def saveImagesFromImageLinks(imageLinks, issueName=""):
-    finalCount = len(imageLinks)
+def saveImagesFromImageLinks(imageLinks, length, issueName=""):
     for imageLink in imageLinks:
-        path = saveImageFromUrl(imageLink, finalCount, issueName)
+        path = saveImageFromUrl(imageLink, length, issueName)
     # path should be the same for all images per folder
     return path
 
-def saveImageFromUrl(url, finalCount, issueName=""):
+def saveImageFromUrl(url, length, issueName=""):
     global COUNTER
-    digits=len(str(finalCount))
+    digits=len(str(length))
```

Comment:
I'd lean away from using the var `length` it's a bit generic considering we're taking the length of length. maybe something like `numberOfImages` ?

Reply:
Updated